### PR TITLE
feat: Improved RangeWidget Styling

### PR DESCRIPTION
**Pull request by ShrimpCryptid**
Resolves https://github.com/allen-cell-animated/agave/issues/103!

I'd recommending starting with the changes to `RangeWidget.h` and `RangeWidget.cpp` when reviewing.

- Renames properties of the RangeWidget to be more clear (`minimum` -> `minBounds`)
- Decoupled the first and second value sliders from min/max
  - This fixes a bug where the first/second sliders could be swapped, resulting in huge and incorrect ranges
- Updates styling for the slider handles based on requests from @TravisKroeker!
- Fixes visual bug where edges of sliders would get cut off
- Updates layouts where range sliders are used (sliders get a larger share of space than the labels)
- Fixed hovering behavior and coloration

Before:
image
After:
image

*Vertical:*
image

**Video Preview:**

https://github.com/allen-cell-animated/agave/assets/30200665/6b01c522-69e3-457b-993f-91c2759b1742

### Notes:
- The color is currently hard-coded for the track background, based on Travis' Figma design. Dan and I talked about how, ideally, this should be a style or theme color for platform compatibility reasons.
image

### CODE REVIEW

File: agave_app/AppearanceSettingsWidget.cpp
Comment (ShrimpCryptid) [inline agave_app/AppearanceSettingsWidget.cpp:1]: I recommend looking at RangeWidget.h first to see the method renaming I did. Almost all the changes in `AppearanceSettingsWidget.cpp` and `loadDialog.cpp` were caused by method renames.

File: agave_app/loadDialog.cpp [hunk 1]
```diff
@@ -84,40 +84,37 @@ LoadDialog::LoadDialog(std::string path, const std::vector<MultiscaleDims>& dims
...
+
+  auto* roiLayout = new QGridLayout();
+  roiLayout->addWidget(new QLabel("X"), 0, 0);
+  roiLayout->addWidget(new QLabel("Y"), 1, 0);
+  roiLayout->addWidget(new QLabel("Z"), 2, 0);
+  roiLayout->addWidget(m_roiX, 0, 1);
+  roiLayout->addWidget(m_roiY, 1, 1);
+  roiLayout->addWidget(m_roiZ, 2, 1);
+  roiLayout->setColumnStretch(0, 1);
+  roiLayout->setColumnStretch(1, 3);
```
Comment (ShrimpCryptid) [inline agave_app/loadDialog.cpp:117]: Replaced with a grid layout to match `AppearanceSettingsWidget.cpp`

File: agave_app/AppearanceSettingsWidget.cpp
Comment (ShrimpCryptid) [inline agave_app/AppearanceSettingsWidget.cpp:1]: I recommend looking at RangeWidget.h first to see the method renaming I did. Almost all the changes in `AppearanceSettingsWidget.cpp` and `loadDialog.cpp` were caused by method renames.

**Review by toloudis — APPROVED** [review]

File: agave_app/RangeWidget.cpp [hunk 2]
```diff
@@ -2,82 +2,193 @@
...
-  QColor c1(m_firstHandleColor);
-  if (m_firstHandleHovered)
-    c1 = c1.darker();
-
-  // Second value handle rect
-  QRectF rt2 = secondTextRect(p);
   QRectF rv2 = secondHandleRect();
-  QColor c2(m_secondHandleColor);
-  if (m_secondHandleHovered)
-    c2 = c2.darker();
```
Comment (ShrimpCryptid) [inline agave_app/RangeWidget.cpp:46]: Moved this section down a bit to make the code cleaner!

**Pull request merged**